.gitignore

```diff
@@ -142,4 +146,5 @@ yarn.lock
 
 # django-webpack-loader
 webpack-stats.json
-backend/uclapi/src
\ No newline at end of file
+backend/uclapi/src
+frontend/webpack.dev.js
```

Comment:
`webpack.dev.js` should actually be committed. I think Anush temporarily put this in, but we should take it out before we merge.

Reply:
Taken this out cheers!